contents/docs/migrate/migrate-between-posthog-instances.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Migrate between PostHog instances
+sidebar: Docs
+showTitle: true
+---
+
+> If you're attempting this migration, feel free to ask questions and provide feedback via the [PostHog Communty Slack workspace](/slack) or [a GitHub issue](https://github.com/PostHog/posthog.com/issues).
+
+## Requirements
+
+- Existing PostHog instance that you can login to (Self-hosted versions need a minimum version of `1.30.0`, if you version is lower you'll need to upgrade to this version first)
+- A new PostHog instance
+
+## Approach
+
+This migration has two parts:
+1. [Migrate your events](#migrate-your-events), this will also create the necessary person, person distinct ID, and related records.
+2. [Migrate your meta data](#migrate-your-meta-data) (projects, dashboards, insights, actions, cohorts, feature flags, experiments, annotations).
+   - Note! It won't move over the following:
+     1. Project API Key. You'll need to replace the API key in your code with the new API key.
+     2. "created by" information. Every object will appear as if it was created by you.
+     3. "created at" information. Every object will appear as if it was created on the time you ran this script.
+
+## Migrate your events
+
+To migrate your events we'll be using the [PostHog Replicator app](/docs/apps/replicator):
```

Comment:
lets not duplicate this info - we can use a snippet and include the same in all places:
1. here
2. in /docs/apps/replicator
3. other places below